TCMSApp/src/client/app/layout/sidebar.service.js

```diff
@@ -0,0 +1,46 @@
+(function () {
```

Comment:
if you call the file sidebar.service it should be one service called sidebar. but here we have two factories. 
let it be one returning functions:
- addProject
- fetchProjects (intentionally had not used verb GET because we shall not do http GET here.)